pkg/antctl/raw/packetcapture/command.go

```diff
@@ -0,0 +1,332 @@
+// Copyright 2025 Antrea Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package packetcapture
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"io"
+	"net"
+	"path"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/utils/ptr"
+
+	"antrea.io/antrea/pkg/antctl/raw"
+	"antrea.io/antrea/pkg/apis/crd/v1alpha1"
+	antrea "antrea.io/antrea/pkg/client/clientset/versioned"
+	"antrea.io/antrea/pkg/util/env"
+)
+
+var (
+	defaultTimeout          = time.Second * 60
+	maxPacketCaptureTimeout = time.Second * 300
+	Command                 *cobra.Command
+	getCopier               = getPodFileCopier
+	defaultFS               = afero.NewOsFs()
+)
+
+type packetCaptureOptions struct {
+	source    string
+	dest      string
+	nowait    bool
+	timeout   time.Duration
+	number    int32
+	flow      string
+	outputDir string
+}
+
+var options = &packetCaptureOptions{}
+
+var packetCaptureExample = `  Start capturing packets from pod1 to pod2, both Pods are in Namespace default
+  $ antctl packetcapture -S pod1 -D pod2
+  Start capturing packets from pod1 in Namespace ns1 to a destination IP
+  $ antctl packetcapture -S ns1/pod1 -D 192.168.123.123
+  Start capturing UDP packets from pod1 to pod2, with destination port 1234
+  $ antctl packetcapture -S pod1 -D pod2 -f udp,udp_dst=1234
+  Save the packets file to a specified directory
+  $ antctl packetcapture -S 192.168.123.123 -D pod2 -f tcp,tcp_dst=80 -o /tmp
+`
+
+func init() {
+	Command = &cobra.Command{
+		Use:     "packetcapture",
+		Short:   "Start capture packets",
+		Long:    "Start capturing packets on the target flow",
+		Aliases: []string{"pc", "packetcaptures"},
+		Example: packetCaptureExample,
+		RunE:    packetCaptureRunE,
+	}
+
+	Command.Flags().StringVarP(&options.source, "source", "S", "", "source of the the PacketCapture: Namespace/Pod, Pod, or IP")
+	Command.Flags().StringVarP(&options.dest, "destination", "D", "", "destination of the PacketCapture: Namespace/Pod, Pod, or IP")
+	Command.Flags().Int32VarP(&options.number, "number", "n", 0, "target number of packets to capture, the capture will stop when it is reached")
```

Comment:
I would recommend using a default value of 1 and making this parameter "optional" as a result. What do you think?

Reply:
i have thought this before but not sure which number is the best default value. but having one > 0 is sure better than 0 i think.